GooglePay/src/main/java/com/braintreepayments/api/PaymentsClientWrapper.java

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api;
+
+import androidx.fragment.app.FragmentActivity;
+
+import com.google.android.gms.wallet.PaymentsClient;
+import com.google.android.gms.wallet.Wallet;
+
+class PaymentsClientWrapper {
```

Comment:
This is the one class that can't be tested with the updated version of the play services wallet library. The static final `Wallet` classes causes errors when tested with Robolectric (required to pass a `FragmentActivity` to the `getPaymentsClient` method, and Powermock (required to mock the static Wallet class).